collect_app/src/main/java/org/odk/collect/android/utilities/PermissionUtils.java

```diff
@@ -41,6 +33,12 @@
 
 public class PermissionUtils {
```

Comment:
I didn't end up having to pull this out to its own module but I like the changes I made to it in preparation so kept them in.